### DISCUSSION TITLE
[@types/color-name] DTS shows named exports, but package has default export

### DISCUSSION THREAD

**Discussion by Yash-Singh1** [DefinitelyTyped/DefinitelyTyped#68128] [Issues with a @types package]
The current TS definitions show named exports for `color-name`, while the real package in fact gives out a default export:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/color-name/index.d.ts

https://github.com/colorjs/color-name/blob/master/index.js#L1

## Workaround

Manual type assertion:

```ts
import * as namedColors from 'color-name';

const allColorNames = Object.keys((namedColors as unknown as { default: typeof import('color-name') }).default)
```

[a comment by an automated account is not shown]

**Comment by dancrumb** [comment]
It's not just `color-name`.

I've hit this on:
- `color-convert`
- `jsonwebtoken`
- `pluralize`

so far.

There seems to be a pattern of `@types/xxx` modules that claim the presence of named exports that are not _actually_ named exports. Things sort of work out fine in CJS projects, but get really unpleasant with ESM projects, because everything looks fine until runtime, which kind of defeats the whole purpose :/
